.github/workflows/merge-pr.yml

```diff
@@ -64,7 +64,7 @@ jobs:
 
   calc-code-coverage:
     name: Merge - Calculate Code Coverage
-    runs-on: [self-hosted, Linux, X64, build, v2]
+    runs-on: ubicloud-standard-8
```

Comment:
This is heavy, needs high memory so falling on standard-8 for code coverage jobs